indico/core/signals/__init__.py

```diff
@@ -14,5 +14,7 @@
 # You should have received a copy of the GNU General Public License
 # along with Indico; if not, see <http://www.gnu.org/licenses/>.
 
-from indico.core.signals import acl, agreements, attachments, category, event, event_management, plugin, users
+from indico.core.signals import (acl, agreements, attachments, category, event, event_management, menu, plugin, users)
 from indico.core.signals.core import *
+
+__all__ = ('acl', 'agreements', 'attachments', 'category', 'event', 'event_management', 'menu', 'plugin', 'users', 'rb')
```

Comment:
Fine for me too. Before 2.0 we could move around some signals anyway (`signals.plugin.template_hook` is not only used for plugins for example)